temp/automatic_ticket_revocations_proposal.md

```diff
@@ -0,0 +1,131 @@
+
+- General Audience
+- Technical Details
+
+The general audience section provides a high-level overview of the proposal.  It includes details suitable for all stakeholders, including those unfamiliar with the inner workings of consensus.
+
+The technical details section contains a more in-depth discussion of the relevant technical aspects, implementation specifics, and rationale for the chosen semantics of the proposal.
+
+# General Audience
+
+## High-Level Overview
+
+This proposal intends to significantly improve the stakeholder user experience around ticket revocations by automatically revoking missed or expired tickets.
+
+Further, it will enable the recovery of funds for users who lost their redeem script for the legacy VSP system (before the release of [vspd](https://blog.decred.org/2020/06/02/A-More-Private-Way-to-Stake/), which removed the need for the redeem script).
+
+## Current State of Ticket Revocations
+
+Missed and expired tickets must be revoked to return the funds used to purchase the ticket to the user.  Missed and expired tickets are defined as follows:
+
+- **Missed tickets** are tickets that do not have a corresponding vote included in the block they are selected.  Tickets are pseudorandomly selected to vote for each block.  The most common cause of missed tickets occurs when a wallet with voting rights for that ticket is not online to cast its vote when a ticket is selected to vote.
+
+- **Expired tickets** are tickets that are not selected to vote before the ticket expiration duration (40,960 blocks, or approximately 142 days, on mainnet).
+
+Revoking a missed or expired ticket requires broadcasting a **ticket revocation transaction**.  In order to manually revoke a ticket today, a user needs to:
+- Open the wallet that has voting rights for the ticket so that the ticket revocation transaction can be signed
+- Revoke the missed or expired ticket(s)
+
+## Automatic Ticket Revocations
+
+This proposal proposes the following changes to enable automatic ticket revocations:
+- Allow tickets to be revoked by anyone instead of requiring a valid signature from the voting address
+	- Ticket revocations will still have to go to the original commitment address in the ticket, as they do today, so this does not allow theft of funds
+	- This would not apply to votes, and therefore voting rights for tickets would remain as they are today
+- Require blocks to contain ticket revocation transactions for all tickets that are missed or expired as of that block
+
+## Consensus Rule Change Cost-Benefit Analysis
+
+When considering changes to the consensus rules, it is paramount to examine both the positive and negative consequences of the proposed changes.  There are nearly always tradeoffs involved in one form or another.  The following two sections provide an analysis in the form of pros and cons accordingly.
+
+### Pros
```

Comment:
I would suggest an addition pro here:

- It simplifies wallet implementations since they will no longer have to include logic to detect the revocation conditions nor create and broadcast revocation transactions in response to them

Reply:
That is a good pro to call out.  Thanks for the suggestion!